vm/src/obj/objint.rs

```diff
@@ -189,7 +188,7 @@ impl PyInt {
 
     #[pymethod(name = "__sizeof__")]
     fn sizeof(&self, _vm: &VirtualMachine) -> usize {
-        mem::size_of::<Self>() as usize
+        self.value.bits() + 7 & !7
```

Comment:
```python
>>>>> (2**1).__sizeof__()
8
>>>>> (2**8).__sizeof__()
16
>>>>> (2**16).__sizeof__()
24
>>>>> (2**32).__sizeof__()
40
```

Reply:
@coolreader18 I think you're right about just not going high enough, I'll test that out

Reply:
sizeof int is sum of size of int object itself + sum of its heap allocated memory.
So, roughly, `mem::sizeof<T> + 4 * self.value.data.len()` is expected.
If current code describes the latter, adding `mem::sizeof` will make the value correct